weaviate/graphql/search_operator.go

```diff
@@ -20,11 +24,22 @@ func (bm25 *BM25SearchOperatorBuilder) WithOperator(operator string) *BM25Search
 // WithMinimumMatch is only relevant for BM25SearchOperatorOr operator.
 func (bm25 *BM25SearchOperatorBuilder) WithMinimumMatch(times int) *BM25SearchOperatorBuilder {
 	if bm25.operator != BM25SearchOperatorAnd {
-		bm25.minimumMatch = times
+		bm25.minimumMatch = int32(times)
 	}
 	return bm25
 }
 
 func (bm25 *BM25SearchOperatorBuilder) build() string {
 	return fmt.Sprintf("{operator:%s minimumOrTokensMatch:%d}", bm25.operator, bm25.minimumMatch)
 }
+
+func (bm25 *BM25SearchOperatorBuilder) togrpc() *pb.SearchOperatorOptions {
+	switch bm25.operator {
+	case BM25SearchOperatorAnd:
+		return &pb.SearchOperatorOptions{Operator: pb.SearchOperatorOptions_OPERATOR_AND, MinimumOrTokensMatch: &bm25.minimumMatch}
+	case BM25SearchOperatorOr:
+		return &pb.SearchOperatorOptions{Operator: pb.SearchOperatorOptions_OPERATOR_OR}
```

Comment:
```suggestion
	case BM25SearchOperatorAnd:
		return &pb.SearchOperatorOptions{Operator: pb.SearchOperatorOptions_OPERATOR_AND}
	case BM25SearchOperatorOr:
		return &pb.SearchOperatorOptions{Operator: pb.SearchOperatorOptions_OPERATOR_OR, MinimumOrTokensMatch: &bm25.minimumMatch}
```

This should be other way around -- minimumOrTokensMatch is [ignored](https://sourcegraph.com/github.com/weaviate/weaviate/-/blob/adapters/repos/db/inverted/bm25_searcher.go?L252-255) for SearchOperator.And; omitting it with SearchOperator.Or would be incorrect